### PR TITLE
Add very basic contribution guidelines

### LINKED ISSUE #91: Add testing/setup documentation for contributors
## Package linking / testing before publishing

- `lerna bootstrap`
- Point out that `lerna bootstrap` is run automatically on `postinstall` (on Travis CI, too)
- `npm link`

## Testing concept

Types of tests:
- Unit tests
- Integration tests
- End-to-end tests

(What they reside, where they occur and when to write which one(s))

## Common pitfalls

- e2e tests: Copy `dependencies` of blocks to test into the `devDependencies` of `webpack`/`webpack2`

### PR DESCRIPTION
**Pull request by sapegin**
Closes #91.

### CODE REVIEW

**Comment by andywer** [comment]
Good idea! I have just one improvement suggestion:

Right now it's more like a DEVSETUP.md. We should at least have a small note to fork and open PRs on github to share the contributions.

How about a structure like

```md
# How to contribute
## Development
## Share
## More information 
```

?

**Comment by andywer** [comment]
Btw, thanks for your PRs and your efforts :)

**Comment by sapegin** [comment]
I usually use [this template](https://github.com/sapegin/dotfiles/blob/master/mrm/contributing/Contributing.md) — tell me what to change and I’ll do it ;-)

**Comment by andywer** [comment]
Let's switch roles then. I will edit the PR tonight and you review my changes :)

**Comment by sapegin** [comment]
OK ;-)

**Comment by andywer** [comment]
Ready to be re-reviewed 😉

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -12,6 +14,23 @@ Run tests:
...
 
+## Code
+
+This repository is a monorepo which means that it contains multiple npm packages.
+
+Running `npm install` in the repository's root directory will install the dependencies of all packages and link the packages that depend on each other.
+
+Running `npm test` in the repository's root directory will run all packages' tests. Run `npm test` in a package to test this package only.
+
+Just edit the code like you always do. Be aware that we automatically lint the code according to [JavaScript standard code style](https://github.com/feross/standard), so stick to the code style or expect the linter to shout at you.
```
Comment (sapegin) [inline CONTRIBUTING.md]: This doesn’t sound very friendly ;-)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -12,6 +14,23 @@ Run tests:
...
+
+When adding a new package you might need to run `npm run postinstall` in the repository's root directory to update the links between packages depending on each other.
+
+
+## Share
+
+Commit your changes, push them and open a pull request. We will review your pull request and eventually merge your valuable contribution back into the mother repository.
```
Comment (sapegin) [inline CONTRIBUTING.md]: Or won’t merge or it won’t be that valuable ;-)

Comment (andywer) [inline CONTRIBUTING.md]: Sure, but I wanted to keep it short ^^

Btw, dropped the "unfriendly" phrase 😉

Comment (sapegin) [inline CONTRIBUTING.md]: I’d mention that `npm test` includes linter. Otherwise looks good.

Comment (andywer) [inline CONTRIBUTING.md]: It's under "Code":
"Just edit the code like you always do. Be aware that we automatically lint the code according to [JavaScript standard code style](https://github.com/feross/standard)."

Comment (sapegin) [inline CONTRIBUTING.md]: I thought it’s about the CI, otherwise how would you lint code on *my* computer? ;-)

Comment (andywer) [inline CONTRIBUTING.md]: "Be aware that `npm test` lints the code ..."?

Comment (sapegin) [inline CONTRIBUTING.md]: At least it’s clear ;-)

**Comment by andywer** [comment]
Ready to be merged?

**Comment by sapegin** [comment]
Yup! :shipit:

**Pull request merged**